core/rawdb/accessors_da_syncer.go

```diff
@@ -0,0 +1,39 @@
+package rawdb
+
+import (
+	"math/big"
+
+	"github.com/scroll-tech/go-ethereum/ethdb"
+	"github.com/scroll-tech/go-ethereum/log"
+)
+
+// WriteDASyncedL1BlockNumber writes the highest synced L1 block number to the database.
+func WriteDASyncedL1BlockNumber(db ethdb.KeyValueWriter, L1BlockNumber uint64) {
+	value := big.NewInt(0).SetUint64(L1BlockNumber).Bytes()
+
+	if err := db.Put(daSyncedL1BlockNumberKey, value); err != nil {
+		log.Crit("Failed to update da synced L1 block number", "err", err)
```

Comment:
```suggestion
		log.Crit("Failed to update DA synced L1 block number", "err", err)
```
Let's capitalize DA everywhere